content/en/docs/Reference/veneers.md

```diff
@@ -161,6 +161,69 @@ schema: olm.bundle
 ```
 </details>
 
+#### Converting from FBC to Basic Veneer
+Operator authors should be able to convert a File-Based Catalog (FBC) to a basic veneer using the below commands:
+
+Converting from File-Based Catalog to a Basic Veneer using `jq`:
+
+```sh
+opm render <catalogRef> -o json | jq 'if (.schema == "olm.bundle") then {schema: .schema, image: .image} else . end'
+```
+
+Example veneer in JSON format after the conversion:
+
+```json
+{
+  "schema": "olm.package",
+  "name": "hello-kubernetes",
+  "defaultChannel": "alpha",
+  "description": "hello-kubernetes"
+}
+{
+  "schema": "olm.channel",
+  "name": "alpha",
+  "package": "hello-kubernetes",
+  "entries": [
+    {
+      "name": "hello-kubernetes.v0.0.1"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "image": "docker.io/test/hello-kubernetes-operator-bundle:v0.0.1"
+}
+```
+
+Converting from File-Based Catalog to a Basic Veneer using `yq`:
+
+```sh
+opm render <catalogRef> -o yaml >> test.yaml
+```
+
+```sh
+yq eval -i 'select(.schema == "olm.bundle") = {"schema": .schema, "image": .image}' test.yaml
+```
```

Comment:
Does it have to be 2 commands? I would have assumed that the output could be piped to `yq` similar to `jq`.

Reply:
Address in [commit](https://github.com/operator-framework/olm-docs/pull/269/commits/9fe177f0ba8eb35354f03c583aee85c5ffc347b8)